apache/incubator-yunikorn-site
# [YUNIKORN-732] Create the docs release for v0.11

What is this PR for?
Create the docs release for v0.11.0 

What type of PR is it?
- [x] Sub-task

What is the Jira issue?
https://issues.apache.org/jira/browse/YUNIKORN-732

How should this be tested?
Use docker or yarn to check website context.

Screenshot
![image](https://user-images.githubusercontent.com/45888688/126054372-e2d347de-e74c-4948-82e8-d63ea7fd29c4.png)
